src/pages/docs/chat/rooms/message-reactions.mdx

```diff
@@ -13,11 +13,66 @@ The reaction `name` represents the reaction itself, for example an emoji. Reacti
 
 The reaction `name` can be any string. Summaries are aggregated based on unique `name` values. UTF-8 emojis are a common use case, but any string can be used as long as they are consistent across all front-ends of your app. Examples of common reaction names are `👍`, `❤️`, `:like:`, `like`, `+1`, and so on. How those are presented to the user is entirely up to the app.
 
+## Types of message reactions <a id="types-of-reactions"/>
+
+Ably Chat supports three types of message reactions. They differ in how they are aggregated and what are the rules for adding and removing them.
+
+| Type | Description | Example | Similar to |
+| ---- | ----------- | ------- | --------- |
+| `Unique` | Users can add a single reaction per message. If they react again, their previous reaction is replaced with the new one. | A user can add a 👍, but adding a ❤️ will replace the 👍. | iMessage, WhatsApp, Facebook Messenger |
+| `Distinct` | Users can add each type of reaction once per message. Multiple different reactions are allowed, but duplicates are not. | A user can add both 👍 and ❤️, but cannot add a second 👍. | Slack |
+| `Multiple` | Users can add unlimited reactions, including duplicates. A count parameter specifies how many reactions to add at once. Each new reaction adds to the total count. | A user can add 10 👍 reactions and 100 ❤️ reactions to the same message. | Claps on Medium |
+
+Note that if adding two identical reactions of type `Distinct`, the second one will be accepted and broadcast as a raw reaction, but it will be ignored in the summary (aggregate). Similarly, when removing a reaction that doesn't exist (of any type), the operation will be accepted and broadcast as a raw reaction, but it will have no effect on the summary.
+
+### Configure the default reaction type <a id="default-type"/>
+
+The default reaction type can be configured at room-level by passing `RoomOptions` when calling `rooms.get`. If nothing is set, the default is `Distinct`.
```

Comment:
This sentence doesn't apply to React - with React it's the options passed to the `ChatRoomProvider`

Reply:
ah good spot, have wrapped in an <If> thingy with a case for react